content/learning-paths/servers-and-cloud-computing/nginx-on-azure/deploy.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Install Nginx on Microsoft Azure Virtual Machine
+weight: 5
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+
+
+## Nginx Installation on Azure Linux 3.0
+
+Install Nginx using `dnf`, start the service, and allow **HTTP/HTTPS** in the firewall. Then access the default welcome page using your virtual machine’s public IP in a browser.
+
+### Install Nginx
+
```

Comment:
in a non-container world, replace 
sudo dnf install -y nginx
with 
sudo apt-get install -y nginx